addons/event/views/event_event_views.xml

```diff
@@ -0,0 +1,345 @@
+<?xml version="1.0"?>
+<odoo>
```

Comment:
As anyway we break the history, we can try to gain an indent level on all view definitions by "shadowing" the data one, like
```
<odoo><data>
    <record>
        ...
    </record>
</data></odoo>
```
That's not a full xml level based indent but that's commonly done to avoid loosing too much indent levels :)